css-env-1/Overview.bs

```diff
@@ -126,6 +126,40 @@ safe area rectangle.
 
 <index type=value for=env()></index>
 
+Maximum Safe area inset variables {#safe-area-max-insets}
+------------------------------------------------------------------
+
+<table dfn-type=value dfn-for="env()">
+	<tr>
+		<th>Name
+		<th>Value
+		<th>Number of dimensions
+	<tr>
+		<td><dfn>safe-area-max-inset-top</dfn>
+		<td><<length>>
+		<td>0 (scalar)
+	<tr>
+		<td><dfn>safe-area-max-inset-right</dfn>
+		<td><<length>>
+		<td>0 (scalar)
+	<tr>
+		<td><dfn>safe-area-max-inset-bottom</dfn>
+		<td><<length>>
+		<td>0 (scalar)
+	<tr>
+		<td><dfn>safe-area-max-inset-left</dfn>
+		<td><<length>>
+		<td>0 (scalar)
+</table>
+
+The maximum safe area insets are four [=environment variables=]
+that are tied to the <a href="#safe-area-insets">safe area inset variables</a>.
+Unlike the safe area inset variables which are dynamic values,
+the maximum safe area insets are static values
+that represent the maximum value of their dynamic counterpart.
```

Comment:
I think you should mention and draw analogy to the small and large viewport: https://www.w3.org/TR/css-values-4/#viewport-variants . In particular, these will be the safe-area-insets when using the large viewport.

This helps differentiate what we mean by "maximum" from things like the values could increase by rotating the device or entering fullscreen.

Reply:
Would adding _“for the current device orientation”_ at the end be sufficient to cover this?

Reply:
Is that sufficient to exclude fullscreen? I didn't think entering fullscreen necessarily counted as a different orientation.

Reply:
How about:

Unlike the safe area inset variables which are dynamic values,
the maximum safe area insets are static values
that represent the maximum value of their dynamic counterpart
when dynamic UA interfaces that are retracted making the [=viewport size=] the [=large viewport size=].

Reply:
Added in 45843b2. Did have to fix some bikeshed-linking, as there is no term `[=viewport size=]` defined in any spec to link to.